### DISCUSSION TITLE
MUI data-grid-pro

### DISCUSSION THREAD

**Discussion by JvilleDjango** [gilbarbara/react-joyride#881] [General]
We are using the data grid pro from Mui and also using your plugin joyride with it. It worked flawlessly before but now when it has to go to a step in a row of the grid  it gets this error and freaks like it cannot find it (index.js:880 Uncaught TypeError: Cannot read properties of undefined (reading 'isSameNode')); Each row has a unique identifying target name. Now sure what happened. Any Ideas?

**Comment by gilbarbara** [comment]
Hey @JvilleDjango 

Can you provide a minimal reproducible example in codesandbox or similar playground?
Most likely an element was removed, so `isSameNode` can't be accessed.
